west.yml

```
@@ -157,7 +157,7 @@ manifest:
       groups:
         - hal
     - name: hal_atmel
-      revision: da767444cce3c1d9ccd6b8a35fd7c67dc82d489c
+      revision: 77d1776bd8d25e405695d7ff84c7043f273406ad
```

As the PR is still open, you need to set it to

```suggestion
      revision: pull/47/head
```

---

So that’s how... :smiley: Thanks !

---

Yes, and now we can approve this PR, but no point in that yet, as it needs another push if the hal PR is merged.